make.ps1

```diff
@@ -1,146 +1,21 @@
-function FindMSBuild
-{
-	$msBuildVersions = @("4.0")
-	foreach ($msBuildVersion in $msBuildVersions)
-	{
-		$key = "HKLM:\SOFTWARE\Microsoft\MSBuild\ToolsVersions\{0}" -f $msBuildVersion
-		$property = Get-ItemProperty $key -ErrorAction SilentlyContinue
-		if ($property -eq $null -or $property.MSBuildToolsPath -eq $null)
-		{
-			continue
-		}
-		$path = Join-Path $property.MSBuildToolsPath -ChildPath "MSBuild.exe"
-		if (Test-Path $path)
-		{
-			return $path
-		}
-	}
-	return $null
-}
-
 if ($args.Length -eq 0)
 {
-	echo "Command list:"
-	echo ""
-	echo "  all             Builds the mod dll."
-	echo "  dependencies    Copies the mod's dependencies into the"
-	echo "                  'OpenRA.Mods.RA2/dependencies' folder."
-	echo ""
-	$command = (Read-Host "Enter command").Split(' ', 2)
-}
-else
-{
-	$command = $args
+	$command = "default"
 }
-
-if ($command -eq "all")
+else if ($args.Length -eq 1)
 {
-	$msBuild = FindMSBuild
-	$msBuildArguments = "/t:Rebuild /nr:false"
-	if ($msBuild -eq $null)
-	{
-		echo "Unable to locate an appropriate version of MSBuild."
-	}
-	else
-	{
-		cd OpenRA.Mods.RA2
-		$proc = Start-Process $msBuild $msBuildArguments -NoNewWindow -PassThru -Wait
-		if ($proc.ExitCode -ne 0)
-		{
-			echo "Build failed. If just the development tools failed to build, try installing Visual Studio. You may also still be able to run the game."
-		}
-		else
-		{
-			echo "Build succeeded."
-		}
-		cd ..
-	}
+    $command = $args
+
+    if ($command == "help" -or $command == "h" -or $command == "--help" -or $command == "-h" -or $command == "/h" -or $command == "/?")
+    {
+        echo "Target list:"
+        echo ""
+        echo "  default   Builds the mod dll."
+        echo "  deps      Copies the mod's dependencies into the"
+        echo "              'OpenRA.Mods.RA2/dependencies' folder."
+        echo ""
+	exit
```

Comment:
Actually, you're right. It shouldn't matter too much as you usually call that using a normal consol window (kinda a separate process) anyway.